components/brave_vpn/common/brave_vpn_utils.cc

```diff
@@ -53,6 +53,7 @@ void RegisterVPNLocalStatePrefs(PrefRegistrySimple* registry) {
 #if BUILDFLAG(IS_MAC)
   registry->RegisterBooleanPref(prefs::kBraveVPNOnDemandEnabled, false);
 #endif
+  registry->RegisterBooleanPref(prefs::kBraveVPNSmartProxyRoutingEnabled, true);
```

Comment:
I think having `false` would be better till we have UI that user toggles it?

Reply:
Updated! 😄